### PR TITLE
Add iShowU Audio Capture

### PR DESCRIPTION
**Pull request by SeekingMeaning**
After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

---

Note: This popup appears during installation and requires an administrator to click Allow in System Preferences:

<details>
<summary>Screenshots</summary>

</details>

### CODE REVIEW

**Comment by vitorgalvao** [comment]
> Note: This popup appears during installation and requires an administrator to click Allow in System Preferences:

Then it needs [the proper `caveat`](https://github.com/Homebrew/homebrew-cask/blob/8cca8f163f9b53573c1c8047df87c0f2d88f4d75/Casks/virtualbox.rb#L62L64).

File: Casks/ishowu-audio-capture.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+cask 'ishowu-audio-capture' do
+  version '1.0.5'
+  sha256 'ed21287187d534554148b6e4331c62a4e6ad2d39dda37e3d9b28de8389d8bd2a'
+
+  url "https://downloads.shinywhitebox.com/driver/iShowUAudioCapture-#{version}.dmg"
+  name 'iShowU Audio Capture'
+  homepage 'https://www.shinywhitebox.com/ishowu'
```
Comment (vitorgalvao) [inline Casks/ishowu-audio-capture.rb:7]: This page is for discontinued apps. I don’t see a page for this specific software.

**Comment by jasongroulx** [comment]
@vitorgalvao Any chance I could help get this merged? I was looking to get this app install with brew myself. What would be required for this to get accepted?

**Comment by vitorgalvao** [comment]
> Any chance I could help get this merged?

It’s discontinued software. We don’t add those to HBC (if they were discontinued after being merged, that’s different).

[Having said that](http://youtu.be/ENHVQ2gslp8), if @SeekingMeaning still wants it, I don’t mind making [an exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/faq/exceptions_notability_threshold.md).

**Comment by jasongroulx** [comment]
😂 All good @vitorgalvao, I just installed it manually and will think of something for my dot-files.

**Comment by vitorgalvao** [comment]
> will think of something for my dot-files.

Making your own [tap](https://github.com/Homebrew/brew/blob/master/docs/How-to-Create-and-Maintain-a-Tap.md) is quite simple.

**Pull request closed without merging**